samples/event-logger/src/components/InputBoard/Filter/Filter.module.less

```diff
@@ -0,0 +1,24 @@
+.eventGroup {
+	display: flex;
+	.pickerItem {
+		display: inline-flex;
+		margin-bottom: 30px;
+		align-items: center;
+	}
+	.switchItem {
+		font-size: 48px;
+		&.small {
+			width: 480px;
+		}
```

Comment:
I think we don't need this.

Reply:
I added such style to keep text aligned even if it becomes 2 lines, not like the picture below.
![eventFilter](https://user-images.githubusercontent.com/77044692/120175234-f17b0980-c240-11eb-9bed-546c3a60565a.png)
Is it okay to remain `&.small`? If not I will fix it to remove this style.

Reply:
Well, I prefer not to set the width explicitly. Because when it's focused it looks like below.
![image](https://user-images.githubusercontent.com/5037429/120284229-04e8ac00-c2f7-11eb-8096-960b3fb7cf22.png)
Let's just remove it. :)

Reply:
It's removed